src/util/expr.cpp

```diff
@@ -38,6 +46,11 @@ void exprt::move_to_operands(exprt &e1, exprt &e2)
   op.back().swap(e2);
 }
 
+/// Move the given arguments to the end of `exprt`'s operands.
+/// The argument is destroyed and mutated to a reference to a nil `irept`.
```

Comment:
:bulb:  Same comment as for the two-argument version. Or could just link to the doc of the two-argument version here.

Reply:
fixed